src/components/CollectionLinkGrid/CollectionLinkGrid.module.scss

```diff
@@ -8,7 +8,7 @@
   grid-template-columns: 1fr;
 
   @media (min-width: 768px) {
-    grid-column: 3/11;
+    grid-column: 1/13;
```

Comment:
A useful value for full span columns is `1 / -1`. -1 counts back from the end column so it would work across all breakpoints.